mod-audit-server/src/test/java/org/folio/dao/InvoiceEventsDaoTest.java

```diff
@@ -0,0 +1,80 @@
+package org.folio.dao;
+
+import static org.folio.utils.EntityUtils.TENANT_ID;
+import static org.folio.utils.EntityUtils.createInvoiceAuditEvent;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import java.util.UUID;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.pgclient.PgException;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import org.folio.CopilotGenerated;
+import org.folio.dao.acquisition.impl.InvoiceEventsDaoImpl;
+import org.folio.rest.jaxrs.model.InvoiceAuditEvent;
+import org.folio.rest.jaxrs.model.InvoiceAuditEventCollection;
+import org.folio.util.PostgresClientFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+
+@CopilotGenerated
+public class InvoiceEventsDaoTest {
+
+  @Spy
+  private PostgresClientFactory postgresClientFactory = new PostgresClientFactory(Vertx.vertx());
+  @InjectMocks
+  InvoiceEventsDaoImpl invoiceEventDao = new InvoiceEventsDaoImpl(postgresClientFactory);
+
+  @BeforeEach
+  public void setUp() {
+    MockitoAnnotations.openMocks(this);
+    invoiceEventDao = new InvoiceEventsDaoImpl(postgresClientFactory);
+  }
+
+  @Test
+  void shouldCreateEventProcessed() {
+    var invoiceAuditEvent = createInvoiceAuditEvent(UUID.randomUUID().toString());
+
+    Future<RowSet<Row>> saveFuture = invoiceEventDao.save(invoiceAuditEvent, TENANT_ID);
+    saveFuture.onComplete(ar -> assertTrue(ar.succeeded()));
```

Comment:
it is not enought just to check if future completed. Need to add verify statement that postgresClientFactory is invoked